alcs-api/src/notification/notification.service.ts

```diff
@@ -0,0 +1,106 @@
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import { Inject, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { IConfig } from 'config';
+import { LessThan, Repository } from 'typeorm';
+import { Application } from '../application/application.entity';
+import { CONFIG_TOKEN } from '../common/config/config.module';
+import { ServiceValidationException } from '../common/exceptions/base.exception';
+import { User } from '../user/user.entity';
+import { NotificationDto } from './notification.dto';
+import { Notification } from './notification.entity';
+
+@Injectable()
+export class NotificationService {
+  constructor(
+    @InjectRepository(Notification)
+    private notificationRepository: Repository<Notification>,
+    @Inject(CONFIG_TOKEN) private config: IConfig,
+    @InjectMapper() private autoMapper: Mapper,
+  ) {}
+
+  async list(userUuid: string) {
+    const notifications = await this.notificationRepository.find({
+      where: {
+        receiver: {
+          uuid: userUuid,
+        },
+      },
+      order: {
+        createdAt: 'DESC',
+      },
+    });
+    return this.mapToDto(notifications);
```

Comment:
Usually, you do not map service results to DTO on the Service layer. DTO responsible more for presentation layer => controllers

Reply:
Done! Moved to controller instead